packages/driver/src/cy/commands/local_storage.ts

```diff
@@ -38,7 +88,13 @@ export default (Commands, Cypress, cy, state) => {
   }
 
   Commands.addAll({
-    clearLocalStorage (keys, options: Partial<Cypress.Loggable> = {}) {
+    getAllLocalStorage: getAllStorage.bind(null, 'localStorage', Cypress),
+    getAllSessionStorage: getAllStorage.bind(null, 'sessionStorage', Cypress),
+
+    clearAllLocalStorage: clearAllStorage.bind(null, 'localStorage', Cypress),
+    clearAllSessionStorage: clearAllStorage.bind(null, 'sessionStorage', Cypress),
+
+    clearLocalStorage (keys, options: Options = {}) {
```

Comment:
I would expect clearLocalStorage to use the same API as clearAllLocalStorage with the current origin specified.

Reply:
I'd rather do that in a separate PR.